assets/css/activation.scss

```diff
@@ -18,13 +18,4 @@
 	li {
 		list-style:none outside;
 	}
-	.submit  {
```

Comment:
What was this rule doing?

Should we build and commit the CSS file for this as well?

Reply:
This rule was being used only by the skip button of the `Install Sensei LMS Pages` notice. It was a very old CSS class and I didn't find the commit with the reason why it was added, but probably to reduce the button relevancy. 🤷‍♂️ Anyway, we'll remove this notice that used it. 😊

And about to commit, after the @yscik nice job with our frontend setup, the idea is not to commit the built files. Now the idea is to generate them while creating the package.

Reply:
> the idea is not to commit the built files

We've talked about this in the past, but maybe things have changed since then. @jom do you know if this is a common practice for modern plugins?

I'm pretty sure Gutenberg and Woo both require a build step. One concern was that external devs might not be expecting to have a required build step, and so this might make it harder for them to contribute. Maybe that isn't a concern anymore?

Reply:
I explored some of this in the Webpack PR #3056. It is common practice in the frontend world. Could be an inconvinence if someone just wants to edit the PHP parts and does not have a `node.js` environment, but I don't think it's unreasonable to require it as part of the development environment today. (Even among Wordpress developers, with Wordpress too having more and more Javascript parts.)

With most of our upcoming work looking to be frontend-heavy, basically all parallel work on a scope like Onboarding would result in extra turns with merging PRs, so this should be definitely worth it.

Reply:
I think if we change something related to this will be another PR. So, for now, I'm merging this PR. But we can continue discussing it ;)

I also checked how WooCommerce is doing it: https://github.com/woocommerce/woocommerce/wiki/Minification-of-SCSS-and-JS

Reply:
Sounds good. I agree that it seems to be becoming more conventional for WP plugins to do this.